src/pages/ValidateConfigFiles.js

```diff
@@ -82,38 +97,31 @@ class EditFile extends Component {
   }
 
   render() {
-    let errorMsgPanel = '';
-    let editPanelCssClass = 'file-editor col-md-12';
-    if (this.props.valid === INVALID) {//TODO - need a max height on the errorMsgPanel
-      errorMsgPanel =
-        <div className="col-md-6 errorMsgPanel">{translate('validate.config.files.msg.invalid')}<br/>
-          <pre className="log">{this.props.invalidMsg}</pre></div>;
-      editPanelCssClass = 'file-editor col-md-6';
-    }
+    //TODO - need a max height on the errorMsgPanel
+    let editPanelCssClass = `file-editor ${this.props.valid === INVALID ? 'col-md-6 verticalLine' : 'col-md-12'}`;
 
     return (
-
-      <div>
+      <div className="validate-config-files">
         <h3>{this.props.file.name}</h3>
         <div className='row'>
-          <div className='col-md-12'>
-            <div className={editPanelCssClass}>
-              <Choose>
-                <When condition={this.state.loading}>
-                  <h3>{translate('loading.pleasewait')}</h3>
-                </When>
-                <Otherwise>
-                  <ValidatingInput
-                    inputValue={this.state.contents}
-                    inputName='fileContents'
-                    inputType='textarea'
-                    inputValidate={YamlValidator}
-                    inputAction={this.handleChange}
-                  />
-                </Otherwise>
-              </Choose>
-            </div>
-            {errorMsgPanel}
+          <div className={editPanelCssClass}>
+            <Choose>
+              <When condition={this.state.loading}>
+                <h3>{translate('loading.pleasewait')}</h3>
+              </When>
+              <Otherwise>
+                <ValidatingInput
+                  inputValue={this.state.contents}
+                  inputName='fileContents'
+                  inputType='textarea'
+                  inputValidate={YamlValidator}
+                  inputAction={this.handleChange}
+                />
+              </Otherwise>
+            </Choose>
+          </div>
+          <div className="col-md-6">
```

Comment:
Feels like this div should also be conditionally rendered? If there isn't an error we just get an empty div.

Reply:
Good point, updated.